lib/ftp.ex

```diff
@@ -33,13 +33,20 @@ defmodule Ftp do
             1 -> Will print messages to and from client
             2 -> Will print messages to and from client, and also all debug messages from the server backend.
     """
-    def start_server(name, ip, port, root_directory, limit_viewable_dirs \\ %{enabled: false, viewable_dirs: []} , username \\ "abc", password \\ "abc", log_file_directory \\ "/var/system/priv/log/ftp", debug \\ 2) do
+    def start_server(name, ip, port, root_directory, options \\ []) do
+        limit_viewable_dirs = options[:limit_viewable_dirs] || %{enabled: false, viewable_dirs: []}
+        username = options[:username] || "abc"
```

Comment:
is this the same as the web UI? i.e. apc ?

Reply:
No, I set it to "abc" on purpose because this is not necessarily an APC module